src/main/resources/config/liquibase/changelog/20170806142424_added_entity_Study.xml

```diff
@@ -61,8 +65,20 @@
                 <constraints nullable="false"/>
             </column>
         </createTable>
+        
+        <!-- New table for storing the list of email address that have received a invitation email -->
+        <createTable tableName="study_invited">
+            <column name="email" type="varchar(255)">
+                <constraints nullable="false" />
+            </column>
+             <column name="studies_id" type="bigint">
```

Comment:
nit: wouldn't an int suffice here?

Reply:
Since the whole project is using BigInt for id, so I just use the same "BigInt"